README.md

```diff
@@ -200,7 +200,12 @@ The HTTP-API exposed by the js-ipfs daemon follows the [`http-api-spec`](https:/
 const repo = <IPFS Repo instance or repo path>
 
 // Create the IPFS node instance
-const node = new IPFS(repo)
+const node = new IPFS({
+  repo: repo,
+  EXPERIMENTAL: {
```

Comment:
Do we have to upcase this? I think `experimental` is enough. What would be good is to print something to the console when this is enabled.